acceptance/tests/i18n/modules/puppet_agent.rb

```diff
@@ -4,6 +4,9 @@
   tag 'audit:high',
       'audit:acceptance'
 
+  skip_test('i18n test module uses deprecated function; update module to resume testing.')
+  # function validate_absolute_path used https://github.com/eputnam/eputnam-i18ndemo/blob/621d06d/manifests/init.pp#L15
```

Comment:
random aside, someone else ran into a similar issue in the `puppet_agent` module https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/628